### DISCUSSION TITLE
including resources in '_posts' path

### DISCUSSION THREAD

**Discussion by canaytore** [mmistakes/minimal-mistakes#3501] [Q&A] [answered]
hello all!

I want to keep my resources (mostly codes) in corresponding/different folders under '_posts' path. You might think I'd generate a folder for each of my blog posts. I saw in the link below that I can store it under 'assets' and 'draft' folders but is it possible to manage my folders accordingly? the reason behind is to better organize my resources

https://jekyllrb.com/docs/posts/

As a more advance request from you, I want it to run only the files under the folder I specified (e.g. ../docs/_posts/xx/print/) on my website.

thanks in advance!

**Comment by mmistakes** [comment, marked as answer]
Core Jekyll can make this difficult. You’ll likely need to use a plug-in like to https://nhoizey.github.io/jekyll-postfiles/
